### PR TITLE
Added 406 unsupported media type response to /image

### PR DESCRIPTION
**Pull request by phouse512**
Based on the conversation from Issue #236 - also includes a basic message to let the user know what happened.

### CODE REVIEW

**Comment by sigmavirus24** [comment]
So, this is good. I'm trying to find if there is a way to communicate to the user what media types we deem acceptable. It's looking like there is no header specifically for this. Perhaps we should just throw it in the body of the response?

**Comment by phouse512** [comment]
Yea I was looking [here] (http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html) at the 406 error code and it says the following:

>Unless it was a HEAD request, the response SHOULD include an entity containing a list of available entity characteristics and location(s) from which the user or user agent can choose the one most appropriate.

Is the 406 only thrown by for bad image types? If so I can just add a list of the supported ones to the data body.

**Comment by sigmavirus24** [comment]
Please stop referencing 2616. 2616 is dead. [This](https://tools.ietf.org/html/rfc7231#section-6.5.6) is what you want.

**Comment by phouse512** [comment]
@sigmavirus24 woops my mistake, I added the body content looks like this: ![Image of terminal] (https://static.dyp.im/F4jIw34s9r/ed1109f08c9bc7af83838ddc9715281e.png)

How does that look?

**Comment by sigmavirus24** [comment]
Is there a reason not to return something better like JSON? Something structured lilke:

```
{
  "message": "Client did not request a supported media type.",
  "accept": [
    "image/webp",
    "image/svg+xml",
    "image/jpeg",
    "image/png",
    "image/*"
  ]
}
```

**Comment by phouse512** [comment]
Good call, it now returns it in that format as json.

File: httpbin/helpers.py [hunk 1]
```diff
@@ -209,6 +217,10 @@ def status_code(code):
                 'x-more-info': 'http://vimeo.com/22053820'
             }
         ),
+        406: dict(data=json.dumps({
```
Comment (sigmavirus24) [inline httpbin/helpers.py]: 2 last nits:

1. Can you update the message to be closer to a sentence like in my example
1. Can you also add the appropriate headers, e.g.,

   ```py
   dict(data=json.dumps({
             # ...
         }),
         headers={'Content-Type': 'application/json'})
   ```

**Comment by phouse512** [comment]
:+1: there we go!

**Comment by sigmavirus24** [comment]
Thanks @phouse512

**Pull request merged**